string srcText          #要翻译的文本
string srcLanguage      #源文本语言
string destLanguage     #目标语言
string type             #使用的翻译开放接口类型，如："baidu"